static_files/css/style.css

```diff
@@ -7,7 +7,34 @@ html {
 }
 
 @import url('https://fonts.googleapis.com/css2?family=Dongle:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Chenla&family=Dongle:wght@300&display=swap');
 
 body{
     font-family: "Dongle", sans-serif;
+    min-height: 100vh;
 }
+
+header{
+    min-height: 50px;
+}
```

Comment:
Why is there minimum height inclusion in the header? As the scope of the task is about implementing the footer, I believe this should be done at another time.